x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -22,6 +23,10 @@ import {
 import { ChartsPluginStart } from '../../../../src/plugins/charts/public';
 import { PluginStartContract as AlertingStart } from '../../alerts/public';
 import { DataPublicPluginStart } from '../../../../src/plugins/data/public';
+import { ConnectorAddFlyoutProps } from './application/sections/action_connector_form/connector_add_flyout';
+import { ConnectorEditFlyoutProps } from './application/sections/action_connector_form/connector_edit_flyout';
```

Comment:
TODO: Check if this breaks lazy load intentions.

Reply:
Looks like further chunks are loaded when opening the flyout, so ✅

Reply:
Yeah, I have the same concern.
I think you can do this to _possibly_ avoid it, but I'm not sure - best to validate using the metrics displayed in CI:
```suggestion
import type { ConnectorAddFlyoutProps } from './application/sections/action_connector_form/connector_add_flyout';
import type { ConnectorEditFlyoutProps } from './application/sections/action_connector_form/connector_edit_flyout';
```